gtfs/spec/en/reference.md

```diff
@@ -234,8 +234,8 @@ Primary key (`trip_id`, `stop_sequence`)
-|  `departure_time` | Time | **Conditionally Required** | Departure time from the stop (defined by `stop_times.stop_id`) for a specific trip (defined by `stop_times.trip_id`).<br><br>If there are not separate times for arrival and departure at a stop, `arrival_time` and `departure_time` should be the same. <br><br>For times occurring after midnight on the service day, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins.<br><br> If exact arrival and departure times (`timepoint=1` or empty) are not available, estimated or interpolated arrival and departure times (`timepoint=0`) should be provided.<br><br>Conditionally Required:<br>- **Required** for `timepoint=1`.<br>- Optional otherwise.| |
+|  `arrival_time` | Time | **Conditionally Required** | Arrival time at the stop (defined by `stop_times.stop_id`) for a specific trip (defined by `stop_times.trip_id`). Time must be either in the HH:MM:SS or H:MM:SS format. The time must be measured in the hours from midnight on the day on which the trip began using the time zone specified by `agency.agency_timezone`. For trips which span midnight in the timezone specified by `agency.agency_timezone`, times after midnight should be greater than 24:00:00. <br><br>If there are not separate times for arrival and departure at a stop, `arrival_time` and `departure_time` should be the same. <br><br> If exact arrival and departure times (`timepoint=1` or empty) are not available, estimated or interpolated arrival and departure times (`timepoint=0`) should be provided.<br><br>Conditionally Required:<br>- **Required** for the first and last stop in a trip (defined by `stop_times.stop_sequence`). <br>- **Required** for `timepoint=1`.<br>- Optional otherwise.|
+|  `departure_time` | Time | **Conditionally Required** | Departure time from the stop (defined by `stop_times.stop_id`) for a specific trip (defined by `stop_times.trip_id`). Time must be either in the HH:MM:SS or H:MM:SS format. The time must be measured in the hours from midnight on the day on which the trip began using the time zone specified by `agency.agency_timezone`. For trips which span midnight in the timezone specified by `agency.agency_timezone`, times after midnight should be greater than 24:00:00.<br><br> If exact arrival and departure times (`timepoint=1` or empty) are not available, estimated or interpolated arrival and departure times (`timepoint=0`) should be provided.<br><br>Conditionally Required:<br>- **Required** for `timepoint=1`.<br>- Optional otherwise.| |
```

Comment:
These definitions should be updated to match the definition of GTFS time. Perhaps instead of repeating the text, they could state that GTFS Time should be used?